src/Runner.Listener/JobDispatcher.cs

```diff
@@ -422,6 +422,10 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                                             lock (_outputLock)
                                             {
                                                 workerOutput.Add(stdout.Data);
```

Comment:
We might need to figure out how to handle the existing logic about stdout from worker.
Today, if the worker printout anything to STDOUT, we will create an issue on the service for it, since we assume anything output from the worker means something busted...

Reply:
you can check the code about how we use `workerOutput` later in the file.